src/main/groovy/org/transmartproject/batch/support/StringUtils.groovy

```diff
@@ -0,0 +1,11 @@
+package org.transmartproject.batch.support
+
+/**
+ * Misc string utils
+ */
+class StringUtils {
+
+    static String escapeForLike(String s, char escapeChar = '\\') {
+        s.replaceAll(/[\\_%]/, "$escapeChar\\\$0")
+    }
```

Comment:
I'd suggest to use SQL escape: `string LIKE pattern [ESCAPE escape-character]`
http://www.postgresql.org/docs/9.0/static/functions-matching.html